src/main/java/se/sundsvall/casedata/api/model/ExtraParameter.java

```diff
@@ -1,18 +1,37 @@
 package se.sundsvall.casedata.api.model;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.List;
 
-import se.sundsvall.casedata.api.model.validation.ValidMapValueSize;
+import jakarta.validation.constraints.NotBlank;
 
 import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
 
 @Data
+@Builder(setterPrefix = "with")
+@Getter
```

Comment:
Isn't `@Data` a shortcut for `@Getter @Setter @RequiredArgsConstructor @ToString @EqualsAndHashCode. `
I.e. some of these annotations can be removed if `@Data` is used